subprojects/docs/src/docs/release/notes.md

```diff
@@ -25,147 +25,20 @@ Switch your build to use Gradle @version@ by updating your wrapper properties:
-The following `Checksum` example task calculates a SHA256 on each given input. Inputs may be updated at any time, for example by adding more inputs
-to the `sources` property or by updating the content of any given input that was previously listed.
-
-```
-import com.google.common.hash.Hashing
-import com.google.common.io.Files
-import org.gradle.api.DefaultTask
-import org.gradle.api.file.ConfigurableFileCollection
-import org.gradle.api.file.DirectoryProperty
-import org.gradle.api.file.FileType
-import org.gradle.api.tasks.InputFiles
-import org.gradle.api.tasks.OutputDirectory
-import org.gradle.api.tasks.PathSensitive
-import org.gradle.api.tasks.PathSensitivity
-import org.gradle.api.tasks.SkipWhenEmpty
-import org.gradle.api.tasks.TaskAction
-import org.gradle.work.ChangeType
-import org.gradle.work.FileChange
-import org.gradle.work.InputChanges
-
-class Checksum extends DefaultTask {
-    @SkipWhenEmpty
-    @PathSensitive(PathSensitivity.NAME_ONLY)
-    @InputFiles
-    ConfigurableFileCollection sources = project.files()
-
-    @OutputDirectory
-    DirectoryProperty outputDirectory = project.objects.directoryProperty()
-
-    @TaskAction
-    void checksum(InputChanges changes) {
-        File outputDir = outputDirectory.get().asFile
-        if (!changes.incremental) {
-            println("Non-incremental changes - cleaning output directory")
-            outputDir.deleteDir()
-            outputDir.mkdirs()
-        }
-        for(FileChange change : changes.getFileChanges(sources)) {
-            File changedFile = change.file
-            if (change.fileType == FileType.FILE) {
-                switch(change.changeType) {
-                    case ChangeType.ADDED: // fall through
-                    case ChangeType.MODIFIED:
-                        outputFileForInput(change).text = hashFileContents(changedFile)
-                        break
-                    case ChangeType.REMOVED:
-                        deleteStaleOutput(outputFileForInput(change))
-                        break
-                }
-            }
-        }
-    }
-
-    private void deleteStaleOutput( File file) {
-        println("Removing old output ${file.name}")
-        file.delete()
-    }
-
-    private File outputFileForInput(FileChange inputFile) {
-        project.file("${outputDirectory.get().asFile}/${inputFile.normalizedPath}.sha256")
-    }
-
-    private String hashFileContents(File file) {
-        println("Hashing ${file.name}")
-        return Files.asByteSource(file).hash(Hashing.sha256()).toString()
+The new [`org.gradle.work.InputChanges`](dsl/org.gradle.work.InputChanges.html) API allows querying for changes to individual input file properties.
+See the [userguide section](userguide/custom_tasks.html#incremental_tasks) for more information how to implement incremental tasks using the new API.
+
+```kotlin
```

Comment:
My understanding was that we explicitly chose not to use syntax highlighting in the release notes. Not sure why, though. Maybe it doesn't work?

Reply:
@aalmiray Any insights on this comment?